rental.rb

```diff
@@ -0,0 +1,17 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
+
```

Comment:
- [ ] Here also, for the same reason I have discussed earlier, we need to have the `attr_accessor` for the `:date, :book, :person`. Something like ``attr_accessor :date, :book, :person``. This enables us to access and set the `:date, :book, :person` properties.